config/hub/samples/clustermanager_v1alpha1_multiclusterworkload.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: clustermanager.onprem.openshift.io/v1alpha1
+kind: MultiClusterWorkload
+metadata:
+  name: multiclusterworkload-sample
+spec:
+  bagOfYAML: |-
+   --- 
+   kind: Deployment
+   apiVersion: extensions/v1beta1
+   metadata:
+     name: nginx-deployment
+   spec:
+     replicas: 3
+     strategy: Recreate
+     selector:
+       matchLabels:
+         deploy: example
+     template:
+       metadata:
+         labels:
+           deploy: example
+       spec:
+         containers:
+           - name: nginx
+             image: nginx:1.7.9
+    ---
+  serviceaccount: gcp-serviceaccount
+  placement:
+  - gcp-cluster1
+  - gcp-cluster2
+  - aws-cluster3   
+status:
+  singleClusterWorkloads:
+  - onprem-system/gcp-cluster1-workload
```

Comment:
What will determine having separate namespaces here? I think for simplicity right now we should assume the same namespace as this resource itself.

Reply:
👍